src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsTests/Nodes/IntersectionNode.tsx

```diff
@@ -0,0 +1,37 @@
+import {Handle, Position} from 'reactflow';
+
+import {Dropdown} from 'antd';
+
+import {IntersectionContainer} from '../SettingsTests.styled';
+
+type IntersectionNodeProps = {
+  data: {
+    withoutHandles?: boolean;
+    showModal: (value: string, group: number | string) => void;
```

Comment:
Why not separate `showTestModal` and `showDelayModal`?

Reply:
To avoid code duplication

Reply:
Duplication is not bad unless it's exactly the same code doing the same stuff with the same data; when it's complex, it needs to be composed, but often it won't be.

I'm not sure if that's the case here though, as I didn't analyze it.

Reply:
If you think I should be separated I can do this

Reply:
Reducing `showModal` modal complexity and decoupling code - nothing more than that 🙂